### ISSUE TITLE
Bug: .sdkmanrc is not loaded when opening a new terminal

### ISSUE THREAD

**Issue by ilopmar** [sdkman/sdkman-cli#779]
**Bug report**

`.sdkmanrc` is loaded when I cd into a directory. Once there, if I open a new terminal in the same directory it is not loaded.
I'm using another similar tool (https://github.com/direnv/direnv) to export environment variables when _cd-ing_ into a directory and it works well when I open a new terminal. Not really sure how that tool or Sdkman detect this.

**To reproduce**

```
cd /tmp
mkdir foo   
cd foo
sdk env init
.sdkmanrc created.

cd .. && cd -

Using java version 8.0.252-open in this shell.

# Open a new terminal from the same directory (CTRL+SHIFT+T in Linux Mint)
# The new terminal is on the same directory but it doesn't load the env
```

![Peek 2020-07-16 10-07](https://user-images.githubusercontent.com/559192/87644339-4c506300-c74c-11ea-8266-648e71340350.gif)

**System info**
- Linux Mint 19.3
- zsh 5.4.2 (x86_64-ubuntu-linux-gnu)
- SDKMAN 5.8.3+506

**Comment by helpermethod** [comment]
Hi! 

Indeed sdkman only reacts to cd'ing into a directory.

I guess Zsh may be able to hook into opening a new terminal and Bash will probably require another hack to simulate Zsh's behaviour 🙄. I'll look into it.

**Comment by ilopmar** [comment]
Thanks for the quick reply. Happy to test any snapshot/beta version :-)

**Comment by helpermethod** [comment]
Dabbled a little bit around and it seems to be a one line fix. PR incoming!

**Comment by marc0der** [comment]
This is now released as version 5.8.5. Thanks again for contributing @helpermethod

**Comment by ilopmar** [comment]
Thank you both for fixing and merging this so fast! ❤️
